### DISCUSSION TITLE
✅ Allow to add a Custom Message when the Loop Grid returns empty results

### DISCUSSION THREAD

**Discussion by PistonPL** [elementor/elementor#23706] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Missing option to define custom message when Loop Grid returns empty result. It's pretty basic functionality..

### Describe the solution you'd like

Add solution similar to old Posts widget

**Comment by nicholaszein** [comment]
Hi, 

this feature was added to Elementor Pro version 3.18.0. You can now customize the "Nothing Found" message in the Loop Grid widget's "Additional Options" that appear once you select your template.

🥳
